public/app/features/profile/ReactProfileWrapper.tsx

```diff
@@ -23,13 +23,15 @@ export const ReactProfileWrapper = () => (
           )}
           <SharedPreferences resourceUri="user" />
           <UserTeams isLoading={states.loadTeams} loadTeams={api.loadTeams} teams={teams} />
-          <UserOrganizations
-            isLoading={states.loadOrgs}
-            setUserOrg={api.setUserOrg}
-            loadOrgs={api.loadOrgs}
-            orgs={orgs}
-            user={user}
-          />
+          {!states.loadUser && (
```

Comment:
shoud this not be be !states.user ?

Reply:
Can you explain more? Because I must refresh page on preference page after navigating from dashboard to see all sections.

Reply:
sorry, my mistake, the naming here was so confusing, loadUser is the name of a function not a boolean